src/components/button/ConnectButtonV1.tsx

```diff
@@ -1,7 +1,8 @@
-import React, { useEffect, useState } from 'react';
-import ConnectButton from './ConnectButton';
 import WalletConnect from '@walletconnect/client';
+// eslint-disable-next-line import/no-unresolved
```

Comment:
This was not resolving properly according to eslint, even when I added it to the package.json.